src/quo2/foundations/colors.cljs

```diff
@@ -185,95 +186,130 @@
 (def info-50-opa-30 (alpha info-50 0.3))
 (def info-50-opa-40 (alpha info-50 0.4))
 
-;; Customization
-(def purple-20 (alpha "#8661C1" 0.2))
-(def purple-50 "#8661C1")
-(def purple-60 (alpha "#8661C1" 0.6))
-(def indigo-20 (alpha "#496289" 0.2))
-(def indigo-50 "#496289")
-(def indigo-60 (alpha "#496289" 0.6))
-(def turquoise-20 (alpha "#448EA2" 0.2))
-(def turquoise-50 "#448EA2")
-(def turquoise-60 (alpha "#448EA2" 0.6))
-(def blue-20 (alpha "#4CB4EF" 0.2))
-(def blue-50 "#4CB4EF")
-(def blue-60 (alpha "#4CB4EF" 0.6))
-(def green-20 (alpha "#5BCC95" 0.2))
-(def green-50 "#5BCC95")
-(def green-60 (alpha "#5BCC95" 0.6))
-(def yellow-20 (alpha "#FFCB53" 0.2))
-(def yellow-50 "#FFCB53")
-(def yellow-60 (alpha "#FFCB53" 0.6))
-(def orange-20 (alpha "#FB8F61" 0.2))
-(def orange-50 "#FB8F61")
-(def orange-60 (alpha "#FB8F61" 0.6))
-(def red-20 (alpha "#F46666" 0.2))
-(def red-50 "#F46666")
-(def red-60 (alpha "#F46666" 0.6))
-(def pink-20 (alpha "#FC7BAB" 0.2))
-(def pink-50 "#FC7BAB")
-(def pink-60 (alpha "#FC7BAB" 0.6))
-(def brown-20 (alpha "#99604D" 0.2))
-(def brown-50 "#99604D")
-(def brown-60 (alpha "#805141" 0.6))
-(def beige-20 (alpha "#CAAE93" 0.2))
-(def beige-50 "#CAAE93")
-(def beige-60 (alpha "#CAAE93" 0.6))
+;;;; Customisation
+
+(def customisation {:purple    {10 "#FAF9FD"
+                                20 "#E7DFF3"
+                                30 "#CABAE4"
+                                40 "#AD94D5"
```

Comment:
Why Map?
There are several color customizations possible, user account, community, wallet etc.. And according to that selected customization color, we have to use different variations of that color for different combinations.

For. ex: (if selected color :green)
Profile: green-50
Cover: green-30-opa-20
community-title color etc.